src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml.cs

```diff
@@ -248,10 +259,11 @@ private void SetComboboxCurrentVersion(string comboboxText, IEnumerable<NuGetVer
             for (int i = 0; i < _versions.Items.Count; i++)
             {
                 DisplayVersion currentItem = _versions.Items[i] as DisplayVersion;
-                if (currentItem != null && (comboboxText == _versions.Items[i].ToString() || _versions.Items[i].ToString() == matchVersion?.ToString()))
+                if (currentItem != null && // null, this represent a bar in UI in the Versions combobox 
+                    (comboboxText.Trim() == currentItem.Version.ToNormalizedString() || matchVersion?.ToString() == currentItem.Version.ToNormalizedString())) // trim extra spaces and compare versions without labels
```

Comment:
```suggestion
string normalizedVersion = currentItem.Version.ToNormalizedString()
                    (comboboxText.Trim() ==  normalizedVersion|| matchVersion?.ToString() == normalizedVersion)) // trim extra spaces and compare versions without labels
```
The idea is to call `currentItem.Version.ToNormalizedString()` method only once.